tests/requirements.txt

```diff
@@ -30,7 +30,7 @@ antsibull-docs==2.21.0
     # via
     #   -c tests/constraints.in
     #   -r tests/requirements.in
-antsibull-docs-parser==1.2.1
+antsibull-docs-parser==1.2.2
```

Comment:
This will fix a few build issues, where bad escaping resulted in invalid RST markup. This should also be backported.